code/app/src/main/AndroidManifest.xml

```diff
@@ -5,7 +5,7 @@
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
-        android:name="com.adobe.marketing.edge.identity.app.EdgeIdentityApplication"
+        android:name="com.adobe.marketing.edge.identity.testapp.EdgeIdentityApplication"
```

Comment:
Change to just `.EdgeIdentityApplication` (note the "."). If you `namespace` is correctly set in the build.gradle.kts then just specifying the class name should resolve and build.